### PR TITLE
 Convert to Swift4.2 syntax & Xcode10 project.

### PR DESCRIPTION
**Pull request by fumiyasac**
I found this master branch is behind in the latest version.
So I convert Swift4.2 syntax & adopt to Xcode10 or above.

__Examples:__

+ `UIApplicationLaunchOptionsKey` → `UIApplication.LaunchOptionsKey`
+ `UIViewContentMode.redraw` → `UIView.ContentMode.redraw`
+ `NSAttributedStringKey` → `NSAttributedString.Key`
+ `kCAMediaTimingFunctionEaseInEaseOut` → `AMediaTimingFunctionName.easeInEaseOut`
+  `UIGestureRecognizerState.ended` → `UIGestureRecognizer.State.ended`

Thanks.

### CODE REVIEW

File: Example/FlexibleSteppedProgressBar/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
             <objects>
-                <viewController id="vXZ-lx-hvc" customClass="ViewController" customModuleProvider="target" sceneMemberID="viewController">
+                <viewController id="vXZ-lx-hvc" customClass="ViewController" customModule="FlexibleSteppedProgressBar_Example" customModuleProvider="target" sceneMemberID="viewController">
                     <layoutGuides>
                         <viewControllerLayoutGuide type="top" id="jyV-Pf-zRb"/>
                         <viewControllerLayoutGuide type="bottom" id="2fi-mo-0CV"/>
                     </layoutGuides>
                     <view key="view" contentMode="scaleToFill" id="kh9-bI-dsS">
-                        <rect key="frame" x="0.0" y="0.0" width="600" height="600"/>
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
```
Comment (amratab) [inline Example/FlexibleSteppedProgressBar/Base.lproj/Main.storyboard]: Is this change required?

File: Example/FlexibleSteppedProgressBar/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="14460.20"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
     <scenes>
         <!--View Controller-->
         <scene sceneID="ufC-wZ-h7g">
             <objects>
-                <viewController id="vXZ-lx-hvc" customClass="ViewController" customModuleProvider="target" sceneMemberID="viewController">
+                <viewController id="vXZ-lx-hvc" customClass="ViewController" customModule="FlexibleSteppedProgressBar_Example" customModuleProvider="target" sceneMemberID="viewController">
```
Comment (amratab) [inline Example/FlexibleSteppedProgressBar/Base.lproj/Main.storyboard]: is this change required?

**Comment by amratab** [comment]
Can you please fix the circle ci failures also by modifying the circleci config file?

File: Example/FlexibleSteppedProgressBar/Base.lproj/Main.storyboard (hunk 1 above)
Comment (fumiyasac) [reply Example/FlexibleSteppedProgressBar/Base.lproj/Main.storyboard]: oh, sorry... `Main.storyboard` changes is not required.
I will revert any `Main.storyboard` changes.

**Comment by fumiyasac** [comment]
@amratab 
I fixed config files. Please check it.

File: Example/FlexibleSteppedProgressBar/Base.lproj/Main.storyboard (hunk 1 above)
Comment (fumiyasac) [reply Example/FlexibleSteppedProgressBar/Base.lproj/Main.storyboard]: I fixed e01ab3b

File: Example/FlexibleSteppedProgressBar/Base.lproj/Main.storyboard (hunk 2 above)
Comment (fumiyasac) [reply Example/FlexibleSteppedProgressBar/Base.lproj/Main.storyboard]: I fixed e01ab3b.
